seed.js

```diff
@@ -1,25 +1,29 @@
 var db = require('./models');
 
-var users = [{
-  user:'xdong',
-  password:'123321',
-  },
-  {user:'kenneth',
-  password:'abccba',
-}
+var users = [
+    {
+      email:'xdong@example.com',
+      password:'123321'
+    },
+    {
+      email:'kenneth@example.com',
+      password:'abccba'
+    }
 ];
 
-db.User.remove({}, function(err, user){
+db.User.remove({}, function(err, u){
   console.log('removed all books');
-    users.forEach(function (userdata) {
+  users.forEach(function (userdata) {
     var user = new db.User({
-      name: userdata.name,
+      email: userdata.email,
```

Comment:
Same here. There is no `name` field on your model! You currently only have email and password.